core/templates/dev/head/components/summary_tile/exploration_summary_tile_directive.html

```diff
@@ -18,7 +18,7 @@ <h3 class="activity-title protractor-test-exp-summary-tile-title" ng-style="isPl
       </h3>
 
       <!-- Note that if this is ng-if instead, the avatar area will not detect a mouseover and the tile will behave incorrectly. -->
-      <div ng-show="isWindowLarge && avatarsList.length > 0" class="exploration-summary-avatars" ng-show="avatarsList.length > 0">
+      <div ng-show="(isWindowLarge && avatarsList.length > 0) || (avatarsList.length > 0)" class="exploration-summary-avatars">
```

Comment:
Are you sure that how you've combined the clauses here makes sense? I think this needs a bit more thought; make sure to understand what the surrounding code does. Don't make these changes cookie-cutter.

Reply:
I tested all the possible boolean conditions using a similar case [here](https://www.w3schools.com/code/tryit.asp?filename=FQKYYF6PVKIS) and found out that the `or` condition needs to be removed so that the two expressions are equivalent. How does this approach sound? Thanks!

Reply:
Yep, the second clause should be removed.